libretroshare/src/libretroshare.pro

```diff
@@ -860,16 +860,25 @@ rs_gxs_trans {
 }
 
 rs_jsonapi {
-    JSONAPI_GENERATOR_SRC=$$system_path($$clean_path($${RS_SRC_PATH}/jsonapi-generator/src/))
-    JSONAPI_GENERATOR_OUT=$$system_path($$clean_path($${RS_BUILD_PATH}/jsonapi-generator/src/))
-    JSONAPI_GENERATOR_EXE=$$system_path($$clean_path($${JSONAPI_GENERATOR_OUT}/jsonapi-generator))
-    DOXIGEN_INPUT_DIRECTORY=$$system_path($$clean_path($${PWD}))
-    DOXIGEN_CONFIG_SRC=$$system_path($$clean_path($${RS_SRC_PATH}/jsonapi-generator/src/jsonapi-generator-doxygen.conf))
-    DOXIGEN_CONFIG_OUT=$$system_path($$clean_path($${JSONAPI_GENERATOR_OUT}/jsonapi-generator-doxygen.conf))
-    WRAPPERS_INCL_FILE=$$system_path($$clean_path($${JSONAPI_GENERATOR_OUT}/jsonapi-includes.inl))
-    WRAPPERS_REG_FILE=$$system_path($$clean_path($${JSONAPI_GENERATOR_OUT}/jsonapi-wrappers.inl))
-
-    restbed.target = $$system_path($$clean_path($${RESTBED_BUILD_PATH}/library/librestbed.a))
+    JSONAPI_GENERATOR_SRC=$$clean_path($${RS_SRC_PATH}/jsonapi-generator/src/)
```

Comment:
Is there an advantage removing `$$system_path()` ? Because I think using both functions together is more cross build system proof

Reply:
shouldn't `$$shell_path()` instead of `$$system_path()` used then instead of removing it?

Reply:
It depends.
Removing it (ie. nothing) gives: C:/msys64/home/blabla
$$shell_path() gives: /C/msys64/home/blabla

The former works better if it's fed to a Windows tool (it's a valid Windows path). The later is needed for some unix tools that do some path manipulation (ie. doxygen) but might break if it's given to a native tool.

For Linux/OSX it doesn't mater, both path should be the same.

Reply:
Thanks for explanation I'll try it on linux to see if it keep working there ;)